toolkit/scripts/imggen.mk

```diff
@@ -11,7 +11,9 @@ assets_files             = $(shell find $(assets_dir))
 imggen_local_repo        = $(MANIFESTS_DIR)/image/local.repo
 imagefetcher_local_repo  = $(MANIFESTS_DIR)/package/local.repo
 imagefetcher_cloned_repo = $(MANIFESTS_DIR)/package/fetcher.repo
-ifeq ($(build_arch),aarch64)
+ifdef CONFIG_FILE
+initrd_config_json       = $(CONFIG_FILE)
+else ifeq ($(build_arch),aarch64)
```

Comment:
The `CONFIG_FILE` is the configuration for the final image we want to build, while `initrd_config_json` is an internal, non-mutable config for the small Mariner image residing on the ISO, which takes care of installing the final image defined inside the `CONFIG_FILE`. In short: these two images are different.

Could you explain in more detail, why you need to a different configuration for the ISO installation?

Reply:
it's actually for the initrd image (not ISO), which will be directly booted from RAM using PXE...not installed.